package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "loopback-android-getting-started",
+  "version": "1.0.0",
+  "description": "LoopBack mobile application to show developers how to use LoopBack Open Node.js Mobile API Middle tier.",
+  "main": "index.js",
+  "scripts": {
+    "test": ""
```

Comment:
No there are not tests to run. The test module is empty ``` "test": ""  ```  in `package.json`
I will test something now and get back to you on that.